Web/Controller/LandingPages.hs

```diff
@@ -17,7 +19,39 @@ instance Controller LandingPagesController where
 
     action ShowLandingPageAction { landingPageId } = do
         landingPage <- fetch landingPageId
+
+        -- Get all the Paragraphs for this Landing Page.
+        -- As we won't have a lot of Paragraphs, we can just fetch them all.
+        paragraphCtas <- query @ParagraphCta
+            |> filterWhere (#landingPageId, landingPageId)
+            |> orderByAsc #weight
+            |> fetch
+
+        paragraphQuotes <- query @ParagraphQuote
+            |> filterWhere (#landingPageId, landingPageId)
+            |> orderByAsc #weight
+            |> fetch
+
+
         render ShowView { .. }
+        where
+            -- We will render each paragraph with it's own renderer, and have a tuple
+            -- with the weight, and the `hsx` result. Then we can sort the list by weight.
+            -- @todo: Why do we need to type hint the paragraph?
+            paragraphCtas' = paragraphCtas
+                    |> fmap (\(paragraph :: ParagraphCta) -> (paragraph.weight, ParagraphCtas.renderParagraph paragraph.title))
+
+            paragraphQuotes' = paragraphQuotes
+                    |> fmap (\(paragraph :: ParagraphQuote) -> (paragraph.weight, ParagraphQuotes.renderParagraph paragraph.title))
+
+            -- @todo: Is there a better way, so render happens inside the Show instead of here.
+            -- Currently without rendering, we can't place all in a single List, as they have different
+            -- types.
+            allParagraphsRendered = merge paragraphCtas' paragraphQuotes'
```

Comment:
This causes 

![image](https://github.com/amitaibu/ihp-landing-page/assets/125707/fad79dc2-0ada-4258-a364-e483de86476e)

1. Why? :smile_cat: 
2. This whole section could probably be improved, so we don't render in the `Controller`, but do it in `Show.hs`